lib/routes/account.js

```diff
@@ -254,6 +259,35 @@ module.exports = function (
             )
         }
 
+        function createKeyFetchToken () {
+          if (wantsKeys(request)) {
+            return password.unwrap(account.wrapWrapKb)
+              .then(
+                function (wrapKb) {
+                  return db.createKeyFetchToken({
+                    uid: account.uid,
+                    kA: account.kA,
+                    wrapKb: wrapKb,
+                    emailVerified: account.emailVerified,
+                    tokenVerificationId: tokenVerificationId,
+                    tokenVerified: false
+                  })
+                }
+              )
+              .then(
+                function (keyFetchTokenData) {
+                  keyFetchToken = keyFetchTokenData.data.toString('hex')
+                }
+              )
+          }
+        }
+
+        function preVerifyAccount () {
+          if (preVerified) {
+            return db.verifyTokens(tokenVerificationId, account);
```

Comment:
Aha, nice, this call-site reads nicely to me (ref https://github.com/mozilla/fxa-auth-server/pull/1232/files#r60862537)